cypress/e2e/v2/homepage.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { ecosystemLinks, resourcesLinks } from "../../fixture/homepage.config";
+
+context("/ on macbook-16", () => {
+  before(() => {
+    // TODO: add mobile
+    cy.viewport("macbook-16");
+  });
+
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  // TODO: fix hover elements issues
+  it.skip("should have Explore section dropdown elements", () => {
+    cy.findByTestId("header-dropDownItem-explore").trigger("mouseover");
+    cy.findByTestId("explore-dropDownItems-$DFI").should("be.visible");
+  });
+
+  // QA-813 - TC2 - Step 2 & 3
+  it("should have Coming soon on Ecosystem and Build sections", () => {
+    cy.get("div[data-testid='header-coming-soon-tag']").should(
+      "have.length",
+      2
+    );
+    cy.get("div[data-testid='header-coming-soon-tag']")
+      .first()
```

Comment:
Why do we need this?

Reply:
changed to unique test ids: `header-coming-soon-tag-Build`